<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KbdStateConfig</class>
 <widget class="QDialog" name="KbdStateConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard state settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="leds">
     <property name="title">
      <string>Lock Indicators</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showCaps">
        <property name="text">
         <string>Show Caps Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showNum">
        <property name="text">
         <string>Show Num Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showScroll">
        <property name="text">
         <string>Show Scroll Lock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="showLayout">
     <property name="title">
      <string>Keyboard Layout Indicator</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="policyLabel">
        <property name="text">
         <string>Switching policy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="switchGlobal">
        <property name="text">
         <string>Global</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="switchWindow">
        <property name="text">
         <string>Window</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QRadioButton" name="switchApplication">
        <property name="text">
         <string>Application</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">modes</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Flags path pattern:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="layoutFlagPattern">
        <property name="toolTip">
         <string>File path pattern for country flags. Must contain &lt;strong&gt;%1&lt;/strong&gt; which is replaced by the two letter ISO country code (lower case).&lt;br&gt;&lt;br&gt;Example: /usr/share/iso-flags-svg/country-squared/%1.svg</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="configureLayouts">
     <property name="text">
      <string>Configure layouts</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>accepted()</signal>
   <receiver>KbdStateConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>rejected()</signal>
   <receiver>KbdStateConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="modes"/>
 </buttongroups>
</ui>
